### PR TITLE
2828 search validation

### PR DESCRIPTION
**Pull request by kkoskelin**
Screen Shot 2019-09-06 at 11 28 43 AM

### CODE REVIEW

File: web-client/pa11y/pa11y-judge.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module.exports = [
   'http://localhost:1234/mock-login?token=judgeArmen&path=/',
+  'http://localhost:1234/mock-login?token=judgeArmen&path=/search',
+  'http://localhost:1234/mock-login?token=judgeArmen&path=/search/no-cases',
```
Comment (rachaelparris) [inline web-client/pa11y/pa11y-judge.js]: Did you mean to use `/search/no-matches` here?

**Review by rachaelparris — APPROVED** [review]

**Comment by rachaelparris** [comment]
I think we may end up needing an entity for validation. We will probably want to check for valid years, and that the year minimum is less than the year maximum.

File: web-client/pa11y/pa11y-judge.js (hunk 1 above)
Comment (kkoskelin) [reply web-client/pa11y/pa11y-judge.js]: correct...

**Comment by kkoskelin** [comment]
Screen Shot 2019-09-06 at 4 05 53 PM

**Pull request merged**